packages/backend/src/config/app.ts

```diff
@@ -38,9 +38,15 @@ dotenv.config({
 export const Config = {
   logLevel: envString('LOG_LEVEL', 'info'),
   enableTelemetry: envBool('ENABLE_TELEMETRY', true),
-  openTelemetryCollectors: envStringArray('OPEN_TELEMETRY_COLLECTOR_URL', [
-    'http://otel-collector-NLB-e3172ff9d2f4bc8a.elb.eu-west-2.amazonaws.com:4317'
-  ]),
+  livenet: envBool('LIVENET', false),
+  openTelemetryCollectors: envStringArray(
+    'OPEN_TELEMETRY_COLLECTOR_URL',
```

Comment:
```suggestion
    'OPEN_TELEMETRY_COLLECTOR_URLS',
```
should it be plural?